shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -394,20 +407,32 @@ void FlutterTizenEngine::SetWindowOrientation(int32_t degree) {
   };
   touch_event_handler_->rotation = degree;
   if (degree == 90 || degree == 270) {
-    renderer_->ResizeWithRotation(geometry.x, geometry.y, height, width,
-                                  degree);
-    SendWindowMetrics(height, width, 0.0);
-  } else {
-    renderer_->ResizeWithRotation(geometry.x, geometry.y, width, height,
-                                  degree);
-    SendWindowMetrics(width, height, 0.0);
+    std::swap(width, height);
   }
+  renderer_->ResizeWithRotation(geometry.x, geometry.y, width, height, degree);
+  // Window position does not change on rotation regardless of it's size.
+  SendWindowMetrics(geometry.x, geometry.y, width, height, 0.0);
 }
 
 void FlutterTizenEngine::OnOrientationChange(int32_t degree) {
   SetWindowOrientation(degree);
 }
 
+#ifndef TIZEN_RENDERER_EVAS_GL
```

Comment:
Why don't you make `SetGeometry` as a member of `TizenRenderer` and remove this ifdef?

The body of `TizenRendererEvasGL::SetGeometry` could be just `FT_UNIMPLEMENTED()`.

Reply:
Great! I could remove a whole lot of `#ifdef`s.